collapsible.js

```diff
@@ -0,0 +1,19 @@
+/** 
+ * Handles making sections of the speedrun route collapsible.
+ */
+
+// Make sections collapsible
+function makeCollapsible() {
+    let coll = document.getElementsByClassName("sectionTitle")
+    for(let i = 0; i < coll.length; i++) {
```

Comment:
is there a reason we're using indexing syntax instead of iterator syntax? (for(x of y))

Reply:
Wrote it this way to maintain browser compatibility for IE. Happy to change to for/of if you'd prefer to stick to ES6.

Reply:
Yeah I'm already using Fetch, CSS3, etc., so backwards compatibility is right out.

Reply:
my targets are firefox 92, chrome (whatever's current), and the mobile equivalents

Reply:
Alright noted for the future. Will leave as is for now since its been merged.